apps/web/package.json

```
@@ -2,8 +2,8 @@
   "name": "@bitwarden/web-vault",
   "version": "2024.12.1",
   "scripts": {
-    "build:oss": "webpack",
-    "build:bit": "webpack -c ../../bitwarden_license/bit-web/webpack.config.js",
+    "build:oss": "NODE_OPTIONS=\"--max-old-space-size=8192\" webpack",
+    "build:bit": "NODE_OPTIONS=\"--max-old-space-size=8192\" webpack -c ../../bitwarden_license/bit-web/webpack.config.js",
```

I'm sorry, we want it for web as well. The CI might not _need_ it but we have developers who use windows. I should have been more clear that I meant it for all commands.

```suggestion
    "build:oss": "cross-env NODE_OPTIONS=\"--max-old-space-size=8192\" webpack",
    "build:bit": "cross-env NODE_OPTIONS=\"--max-old-space-size=8192\" webpack -c ../../bitwarden_license/bit-web/webpack.config.js",
```